### ISSUE TITLE
Not working with your sample code

### ISSUE THREAD

**Issue by amoshsueh** [TimmysApp/ScrollViewStyle#1]
Hi, I found this library from [stackoverflow](https://stackoverflow.com/a/73706932/3203181).

But it's not working with below error:
`Overriding non-open instance method outside of its defining module`

Can you help to check this issue? Thank you very much.

<img width="1427" alt="截圖 2023-05-18 上午12 12 46" src="https://github.com/TimmysApp/ScrollViewStyle/assets/8340691/c8bba3bc-4281-4cd6-8f26-c2077f2016ec">

**Comment by adamvnovak** [comment]
I'm having the same issue

**Comment by SwiftyJoeyy** [comment]
Hey @adamvnovak @amoshsueh, I sincerely apologize for the delay. The issue has been resolved in the latest version 2.0.0. 

**New usage example:**

 1. Add a property to track the state of the `ScrollView`
    ```
    @ScrollState var state
    ```
 2. Use the `scrollViewStyle` modifier
    ```
    .scrollViewStyle(.defaultStyle($state))
    ```
TestView

```
struct TestView: View {
    @ScrollState var state
    var body: some View {
        ScrollView {
            ...
        }.scrollViewStyle(.defaultStyle($state))
        .onChange(of: state.isDragging) { newValue in
            print(newValue)
        }
    }
}
```
